### DISCUSSION TITLE
Reinstall with so-setup fails

### DISCUSSION THREAD

**Discussion by p3rmAn3nTchA0S** [Security-Onion-Solutions/securityonion#8162] [Unsupported Versions] [answered]
Hello

I installed Security Onion successfully in my machine using the official ISO. However because of some configuration changes that I made and I forgot, I wanted to reinstall it but without putting a USB ISO in my machine. Therefore Im trying to reinstall it using the so-setup.

I tried several approaches of so-setup which includes:
`so-setup iso`
`so-setup-network`

I even tried the "Airgap" mode just in case. Unfortunately installation is never succesful and I am receiving errors as related in the following post: https://github.com/Security-Onion-Solutions/securityonion/discussions/7580#discussion-3941395

Im putting the same configuration as before in my attempts. So could someone help me or at least guide me what am I doing wrong?

Thank you in advance :)

**Comment by weslambert** [comment, marked as answer]
It might be quicker and easier to just reinstall from the ISO.
